contracts/external/traders/Expiry.sol

```diff
@@ -101,6 +116,18 @@ contract Expiry is
         g_expiryRampTime = newExpiryRampTime;
     }
 
+    // ============ Approval Functions ============
+
+    function approveSender(
```

Comment:
I wonder if we should also make one that can be called via the call action on the protocol, that might be useful later